### DISCUSSION TITLE
zhishixingqiu/

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#124] [Announcements]
# zhishixingqiu/

沉默王二的编程知识星球-Java程序员进阶之路-第二次介绍，这是一个 Java 学习指南 + 编程实战的私密圈子，你可以向二哥提问、帮你制定学习计划、跟着二哥一起做实战项目，冲冲冲。

https://tobebetterjavaer.com/zhishixingqiu/

**Comment by jackDcSung** [comment]
可以給我VX嗎 這個頁面看不到碼

**Comment by lutu1123** [comment]
优惠卷还有吗，显示领光了
